config/deploy.rb

```diff
@@ -46,8 +46,7 @@
 
 task :restart_workers do
   on roles(:worker) do
-    sudo 'stop workers || true'
-    sudo 'start workers'
+    sudo 'systemctl restart rails-assets-build.service'
```

Comment:
Does `systemctl restart` cause the workers to start if they aren't already running? That was the purpose of the previous script.